### ISSUE TITLE
S3 HTML Template does not provide device specific warnings

### ISSUE THREAD

**Issue by ChrisChares** [fastlane/fastlane#5774]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

S3 Action generates a HTML page from which a user can install the `.ipa`.  However it doesn't take into account what specific devices this `.ipa` supports.  For instance, if the app is iPad only and you open the page on a desktop, it says 
> Please open this page on your iPhone!

If you then open the page on your iPhone, it will let you attempt the install.  Unfortunately the system will allow this, and then fail the install with an generic warning instead of just telling the user the app is iPad only.  

### Configuration Files

Fastfile:
```ruby
default_platform :ios

platform :ios do

  before_all do
    ensure_git_status_clean
  end

  def bump
    increment_build_number
    commit_version_bump
  end

  desc "Build and sign for enterprise (beta) release"
  lane :beta do

    bump
    
    # Build
    gym(
      scheme: "ro",
      output_name: "ro.ipa",
      configuration: "Enterprise",
      codesigning_identity: "iPhone Distribution: 303Software inc",
      provisioning_profile_path: "./fastlane/certs/RO_Enterprise.mobileprovision",
      use_legacy_build_api: true
    )

    # Deploy
    s3(
      access_key: ENV['S3_ACCESS_KEY'],               # Required from user.
      secret_access_key: ENV['S3_SECRET_ACCESS_KEY'], # Required from user.
      bucket: 's3.303apps.com/ro',                       # Required from user.
      ipa: 'build/ro.ipa',                             # Optional is you use `ipa` to build
      dsym: 'build/ro.app.dSYM.zip',                   # Optional is you use `ipa` to build
      upload_metadata: true,                          # Upload version.json, plist and HTML. Set to false to skip uploading of these files.
    )

  end
end
```

### Environment

fastlane version (run `fastlane -v`):
fastlane 1.100.0
Do you use bundler to execute fastlane (i.e. `bundle exec fastlane`)?
No
Do you use a Ruby environment manager (e.g. `chruby`, `rbenv`, `rvm`)?
No

**Comment by ChrisChares** [comment]
I propose (implemented in PR #5775) the following behavior.

+ When viewed from a desktop, the default HTML should tell you what type device you need to install on.
+ When viewing an iPad only app on an iPhone, you should not be able to install and the warning should tell you to `Please open this page on your iPad!`

**Comment by ChrisChares** [comment]
PR merged.  Thanks @ohwutup
